src/app/core/session/session-service/remote-session.ts

```diff
@@ -125,6 +142,29 @@ export class RemoteSession {
         withCredentials: true,
       })
       .toPromise();
-    this.connectionState.setState(ConnectionState.DISCONNECTED);
+    this.currentDBUser = undefined;
+    this.getConnectionState().setState(ConnectionState.DISCONNECTED);
+    this.getLoginState().setState(LoginState.LOGGED_OUT);
+  }
+
+  getCurrentDBUser(): DatabaseUser {
+    return this.currentDBUser;
+  }
+
+  checkPassword(username: string, password: string): boolean {
+    // Cannot be checked against CouchDB due to cookie-auth
+    return false;
```

Comment:
throw error (or log warning) to avoid accidental use here and in getCurrentUser?